Tasks/Common/webdeployment-common/xmlvariablesubstitutionutility.ts

```diff
@@ -9,13 +9,14 @@ var fileEncoding = require('./fileencoding.js');
 
 export async function substituteAppSettingsVariables(folderPath) {
     var configFiles = tl.glob(folderPath + "/**/*.config");
-    var tags = ["applicationSettings", "appSettings", "connectionStrings", "configSections"];
+    var variableMap = varUtility.getVariableMap();
+    var tags = ["applicationSettings", "appSettings", "connectionStrings", "configSections", "system.webServer"];
```

Comment:
Is it fine to add system.webserver section? What does it mean for other App types and default behaviour in transformation?